### ISSUE TITLE
Cannot paste in find widget due to `webview` paste handler

### ISSUE THREAD

**Issue by alexdima** [microsoft/vscode#127881]
I was reviewing a PR and typed in a comment and generally self-hosted. (I might have opened a webview to see the PR but the focus is not in a webview).

* click in the find widget
* paste
* observe that the paste is handled by the `webview` paste handler

https://user-images.githubusercontent.com/5047891/124288592-a579f800-db51-11eb-9615-a48bf0d6ef42.mp4

```
[2021-07-02 16:21:32.655] [renderer3] [info] [KeybindingService]: + Storing single modifier for possible chord ctrl.
[2021-07-02 16:21:32.960] [renderer3] [info] [KeybindingService]: + Clearing single modifier due to 300ms elapsed.
[2021-07-02 16:21:38.362] [renderer3] [trace] CommandService#executeCommand setContext
[2021-07-02 16:21:39.047] [renderer3] [info] [KeybindingService]: / Received  keydown event - modifiers: [ctrl], code: ControlLeft, keyCode: 17, key: Control
[2021-07-02 16:21:39.048] [renderer3] [info] [KeybindingService]: | Converted keydown event - modifiers: [ctrl], code: ControlLeft, keyCode: 5 ('Ctrl')
[2021-07-02 16:21:39.048] [renderer3] [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
[2021-07-02 16:21:39.512] [renderer3] [info] [KeybindingService]: | Resolving ctrl+V
[2021-07-02 16:21:39.512] [renderer3] [info] [KeybindingService]: \ From 4 keybinding entries, matched editor.action.clipboardPasteAction, when: no when condition, source: built-in.
[2021-07-02 16:21:39.512] [renderer3] [info] [KeybindingService]: / Received  keydown event - modifiers: [ctrl], code: KeyV, keyCode: 86, key: v
[2021-07-02 16:21:39.512] [renderer3] [info] [KeybindingService]: | Converted keydown event - modifiers: [ctrl], code: KeyV, keyCode: 52 ('V')
[2021-07-02 16:21:39.512] [renderer3] [info] [KeybindingService]: | Resolving ctrl+V
[2021-07-02 16:21:39.512] [renderer3] [info] [KeybindingService]: \ From 4 keybinding entries, matched editor.action.clipboardPasteAction, when: no when condition, source: built-in.
[2021-07-02 16:21:39.512] [renderer3] [trace] KeybindingService#dispatch Ctrl+V editor.action.clipboardPasteAction
[2021-07-02 16:21:39.512] [renderer3] [trace] CommandService#executeCommand editor.action.clipboardPasteAction
[2021-07-02 16:21:39.513] [renderer3] [trace] Command 'editor.action.clipboardPasteAction' was handled by 'webview'.
```

The problematic registration comes from here -- https://github.com/microsoft/vscode/blob/fbdcbeece36294549e85eb5c803638c9f4ddfa35/src/vs/workbench/contrib/webview/browser/webview.contribution.ts#L18

I had to reload window to get out of that state. Giving important because pasting is quite important to be broken.

**Comment by bpasero** [comment]
I have seen similar issues that I could never explain where suddenly basic editor commands would not work anymore. Not sure if related to this one.

**Comment by IllusionMH** [comment]
Similar to #126716 and #126458

**Comment by joaomoreno** [comment]
Great findings!

**Comment by rangav** [comment]
Due to this bug some of the key functionality in [Thunder Client](https://marketplace.visualstudio.com/items?itemName=rangav.vscode-thunder-client) extension not working

I am hoping vscode fix this ASAP.

**Comment by mjbvz** [comment]
I believe this was fixed already in #126458

Please let me know if you still see this issue and I'll take a  look
